qa/config.template.json

```diff
@@ -0,0 +1,9 @@
+{
+  "config": {
+    "languages": {
+      "%s": {
+        "mass_threshold": 20
```

Comment:
Should we be overriding the mass_threshold here?

Reply:
Nope, I don't think we need to. Updated